lib/structures/CommandInteraction.js

```diff
@@ -105,7 +108,7 @@ class CommandInteraction extends Interaction {
                 this.member = this.channel.guild.members.update(data.member, this.channel.guild);
             } else {
                 const guild = this._client.guilds.get(data.guild_id);
-                this.member = guild.members.update(data.member, guild);
+                this.member = guild?.members?.update(data.member, guild);
```

Comment:
Not backwards compatible

Reply:
What do you propose since guild and member will be undefined on user-installable commands?

Reply:
CNR
![image](https://github.com/user-attachments/assets/3734c156-992e-4e37-9e7c-d21e83a2f3d4)

Reply:
@bsian03 could you review this again. I think the approach I took is a bit weird.

Reply:
Hey, I am going to resolve this conversation. Could you submit a new review if changes are needed? Thanks!